acceptance/tests/windows/PUP-9719_windows_system_first_pa_run.rb

```diff
@@ -0,0 +1,52 @@
+test_name "PUP-9719 Windows First Agent run as system sets cache file permissions correctly"
```

Comment:
you can wrap the `test_name` in a `do..end` block

Reply:
Done